src/main/groovy/com/inxmail/clicktracker/controller/ApiController.groovy

```diff
@@ -0,0 +1,39 @@
+package com.inxmail.clicktracker.controller
+
+import com.inxmail.clicktracker.entity.PostUrl
+import com.inxmail.clicktracker.entity.Url
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.ResponseBody
+
+/**
+ * Created by fewi on 18.06.2015.
+ */
```

Comment:
Please remove these comments on all files. Version control will remember just fine who is the original author of a file.

If you are not familiar with the book Clean Code by Uncle Bob I would highly recommend it to you. We have a copy in the library. Chapter 4 about Comments is interesting in this context.